src/components/info/qa-entry.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import Markdown from "react-markdown";
+import ChevronDown from "../icons/chevron-down";
+import ChevronRight from "../icons/chevron-right";
+
+interface QaEntryProps {
+  question: string;
+  answer: string;
+  key: string;
+  isLast: boolean;
+  isInitiallyExpanded: boolean;
+}
+
+const QaEntry: React.FC<QaEntryProps> = ({
+  question,
+  answer,
+  key,
+  isLast,
+  isInitiallyExpanded,
+}) => {
+  const [isExpanded, setIsExpanded] = useState(isInitiallyExpanded);
+  return (
+    <div key={key} className={`w-full ${!isLast && "border-b-2"} py-4`}>
+      <button
+        className="flex w-full flex-row justify-between text-xl"
+        onClick={() => {
+          setIsExpanded(!isExpanded);
+        }}
+      >
+        <div>{question}</div>
+        <div>
+          {isExpanded ? (
+            <ChevronDown></ChevronDown>
```

Comment:
these should both be in gdk-blue (color defined here https://github.com/technologiestiftung/giessdenkiez-de/pull/718)